Inventory and inventory food

SaeqaSultani opened the pull request:
Closes #10 
Closes #11 
- Inventories list:
    1 - Should display a list of inventories created by the logged-in user as in the wireframe.
     2 - Should lead to inventory details.
     3 - If the user is the owner of the inventory, should allow to delete it.
- Fix linters

SaeqaSultani committed:
Create controllers for Inventories and InventoryFoods

SaeqaSultani committed:
Add description column to the inventories table

SaeqaSultani committed:
create index method for invetories_controller

SaeqaSultani committed:
Add create action and destroy action to the inventories_controller

SaeqaSultani committed:
create destroy and show method for inventory_controller

SaeqaSultani committed:
Fix linters

SaeqaSultani committed:
Merge branch 'dev' into Inventory_and_InventoryFood

Wahaj-Ali reviewed (changes requested):
## Status: Changes Required
Hi @SaeqaSultani 👋 

![so clsoe1](https://user-images.githubusercontent.com/111431787/233445009-23589132-03fa-4b84-b89c-42c8409a319d.gif)


## Highlights

Your design does not match the wireframe ❌ 
- I think you need to change the view for the inventories index and show files in order to match the design from the wireframe.

## Your design for Inventories page:
![inventories index](https://user-images.githubusercontent.com/111431787/233446757-5df37025-d2a2-43c8-b7c3-628a4e03c22a.jpg)
## Wireframe design:
![inventory_index](https://user-images.githubusercontent.com/111431787/233446891-c7f9d773-3b1d-457d-bd5d-8c0e012bf974.png)

## Your individual inventory page(show):
![inventories show](https://user-images.githubusercontent.com/111431787/233447074-dbaf822c-8c7e-4b83-b493-16dba0ec8c7f.jpg)
## Wireframe design:

![inventory_food](https://user-images.githubusercontent.com/111431787/233447170-32ecef80-b7a9-4107-b440-32375a0774be.png)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Wahaj-Ali committed:
Update the inventories page.

Wahaj-Ali committed:
Add inventory foods page and all its functionalities.

In this commit
- Set up the controller for inventory foods.
- Edit the food controller to modify the destroy method.

Wahaj-Ali committed:
Update inventories_controller.rb

Wahaj-Ali committed:
Update index.html.erb

Add access to inventory food  path from the inventory index page.

Wahaj-Ali committed:
Update index.html.erb

- Add views to the inventory food index page.

Wahaj-Ali committed:
Delete new.html.erb

Wahaj-Ali committed:
Add a form to add new food to the inventory food.

Wahaj-Ali committed:
Remove linter errors.

Wahaj-Ali committed:
Update inventories_controller.rb
